drools/kogito-wiring-static/pom.xml

```
@@ -10,11 +10,11 @@
      <version>2.0.0-SNAPSHOT</version>
    </parent>
 
-    <artifactId>drools-core-static</artifactId>
+    <artifactId>kogito-wiring-static</artifactId>
 
     <name>Kogito :: Drools :: Drools Core Static</name>
```

Can you add a `description`?
```suggestion
    <name>Kogito :: Drools :: Drools Wiring Static</name>
```